calico-enterprise/operations/monitor/metrics/recommended-metrics.mdx

```diff
@@ -0,0 +1,576 @@
+
+### Typha
+
+Typha is a caching datastore proxy that sits between calico-nodes and Kubernetes API Server. Its primary function is to allow for increased cluster scale by reducing the load on Kubernetes API Server. Without Typha, large clusters (200+ nodes) would need a considerable amount of memory to correspond to the continuous watches and requests from calico-nodes running in the cluster.
+
+Typha maintains a single datastore connection on behalf of all of its clients (processes running in the calico-node pods, with Felix being Typhas’ main client). Typha watches for node, pod, network policy, bgp configuration, and other events on the Kubernetes API Server, caches and deduplicates this data, and fans out these events to its clients.
+
+### Felix
+
+Felix is a component of calico-node and is responsible for {{prodname}} network policy.
+Felix must be continuously in sync with the datastore to ensure the correct set of policies are applied to the node it is running on.
+
+![Typha-felix](/img/calico-enterprise/typha-felix.png)
+
+### About metrics
+
+Each {{prodname}} component that you want to connect to Prometheus for endpoint metrics has its own configuration (bgp, license, policy, felix, and typha).
+
+Note that Felix is a separate application with metric endpoints, its own core metrics to monitor itself, and a separate port for a second policy metric endpoint.
+
+## Metrics
+
+This section provides metrics recommendations for maintaining optimal cluster operations. Note the following:
+
+- Threshold values for each metric depend on the cluster size and churn rate.
+- Threshold recommendations are provided where possible, but because each cluster is different, and metrics can depend on cluster churn rate and scale. We recommend that you baseline the cluster to establish numbers that represent normal figures for your cluster.
+- Metrics that start increasing rapidly from the baseline set need attention.
+
+**Policy**
+- [{{prodname}} policy metrics](#calico-enterprise-policy-metrics)
+
+**Typha**
+- [Typha general metrics](#typha-general-metrics)
+- [Typha cluster mesh metrics](#typha-cluster-mesh-metrics)
+- [Typha client metrics](#typha-client-metrics)
+- [Typha cache internals](#typha-cache-internals)
+- [Typha snapshot details](#typha-snapshot-details)
+
+**Felix**
+- [Felix cluster state metrics](#felix-cluster-state-metrics)
+- [Felix error metrics](#felix-error-metrics)
+- [Felix time-based metrics](#felix-time-based-metrics)
+
+## Policy metrics
+
+### Denied traffic
+
+| Denied traffic                   |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | <code>calico_denied_packets<br>calico_denied_bytes</br></code> |
```

Comment:
I think your problem lies entirely with the line breaks.
* In regular HTML, it's valid to use just `<br>`.
* Our Docusaurus setup is stricter. You must explicitly close the tag like this: `<br />`.
* You can't mark up content as a line break: `<br>calico_denied_bytes</br>` doesn't work. Either you have two line breaks on either side -- `<br />` -- or you do something else.

Reply:
Fixed.